plugins/gradle/gradle.plugin.zsh

```diff
@@ -60,6 +60,35 @@ _gradle_does_task_list_need_generating () {
   [[ ! -f .gradletasknamecache ]] || [[ build.gradle -nt .gradletasknamecache ]]
 }
 
+##############
+# Parse the tasks from `gradle(w) tasks --all` into .gradletasknamecache
+# All lines in the output from gradle(w) that are between /^-+$/ and /^\s*$/
+# are considered to be tasks. If and when gradle adds support for listing tasks
+# for programmatic parsing, this method can be deprecated.
+##############
+_gradle_parse_tasks_to_gradletasknamecache () {
```

Comment:
I would also change this function name to `_gradle_parse_tasks`. See https://github.com/robbyrussell/oh-my-zsh/pull/3641/files/212047e799ca49e4910489db4e68158bc07a7ca8..e9756bab915d3646e884fc67357a091e7f622565#r78707044.